CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# Contribute
+
+CARTO VL platform is an open-source library. We are more than happy to receive your contributions to the code and the documentation as well.
+
+## Filling a ticket
+
+If you want to open a new issue in our repository, please follow these instructions:
+
+- Add a descriptive title.
+- Write a good description, answering the following questions:
+    - what is happening?
+    - what should happen?
+    - how to reproduce the issue. Remember to include Browser, OS and carto-vl version.
+- Try to add an example showing the problem (using JSFiddle, JSBin or a just code snippet).
+
+
+## Contributing code
+
+Best part of open source, collaborate in CARTO VL code! We like hearing from you, so if you have any bug fixed, or a new feature ready to be merged. Those are the steps you should follow:
```

Comment:
Change to "The best part of open source: collaborate directly with our CARTO VL code! If you have fixed a bug or have a new functionality on your hands, we'd love to see it! These are the steps you should follow:"